src/t2iapi/context/service.proto

```diff
@@ -90,22 +80,22 @@ service ContextService {
       returns (BasicResponse);
 
   /*
-  Remove at least one pm:Identification element for the pm:AbstractContextState/@Handle.
-  The manipulated state shall be persistent until a next manipulation call. If the device is not able to maintain
-  the static state, it shall return RESULT_NOT_SUPPORTED.
+  For the Context with the provided descriptor handle perform the following transition:
+  - create a new associated pm:AbstractContextState instance with at least one unique pm:Identification element
+    for the Context with the given descriptor handle
+  - remove at least one pm:Identification element for the previously created pm:AbstractContextState.
  */
-  rpc RemoveIdentificationOfContextState(
-  t2iapi.context.ContextStateHandleRequest)
-      returns (BasicResponse);
+  rpc TransitionFromCreateStateWithIdentificationToRemoveIdentification (BasicHandleRequest)
+      returns (CreateContextStateWithAssociationResponse);
 
   /*
-  Change at least one pm:Identification element for the pm:AbstractContextState/@Handle.
-  The manipulated state shall be persistent until a next manipulation call. If the device is not able to maintain
-  the static state, it shall return RESULT_NOT_SUPPORTED.
+  For the Context with the provided descriptor handle perform the following transition:
+  - create a new associated pm:AbstractContextState instance with at least one unique pm:Identification element
+    for the Context with the given descriptor handle
+  - change at least one pm:Identification element for the previously created pm:AbstractContextState.
```

Comment:
Does this require changing an already present identification, or does adding a new element also pass as change?

Reply:
it shall change an already present identification. So I will update the description.